client/Components/Decks/ImportDeckModal.jsx

```diff
@@ -4,7 +4,7 @@ import {
     useGetFactionsQuery,
     useGetPacksQuery
 } from '../../redux/middleware/api';
-import { processThronesDbDeckText } from './DeckHelper';
+import { processThronesDbDeckText, processPlainDeckText } from './DeckHelper';
```

Comment:
I think I would prefer these be a single function, and then within that function it does the ThronesDB check followed by the plain text check, but not a major issue more just preference to keep logic consistent & simple :)